Gemfile

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-source 'https://rubygems.org'
+source 'https://www.rubygems.org'
```

Comment:
Why?

Reply:
Yesterday, I had an issue with `bundle install`, and got a suggestion to change the source, but right now it works just fine without this. I will change it back right away.